src/modules/info-panel/info-panel.tag.html

```diff
@@ -1,4 +1,4 @@
-<info-panel class="viewer-info" style="height:60%;">
+<info-panel class="viewer-info { 'is-maximized': isMaximized }" id="info-panel" aria-expanded="true">
```

Comment:
I would move the `id="info-panel"` and `aria-expanded="true"` to [the info-panel instance in in `app.tag.html`](https://github.com/voorhoede/demo-viewer/blob/ee033c9ba880339345621088758d72c68a515799/src/modules/app/app.tag.html#L20) as that's the context in which these attributes apply (related to button in that same module).